src/erlcloud_ses.erl

```diff
@@ -794,8 +948,23 @@ encode_params([{source, Source} | T], Acc) when is_list(Source); is_binary(Sourc
     encode_params(T, [{"Source", Source} | Acc]);
 encode_params([{subject, Subject} | T], Acc) ->
     encode_params(T, encode_content("Message.Subject", Subject, Acc));
+encode_params([{template_name, TemplateName} | T], Acc) when is_list(TemplateName); is_binary(TemplateName) ->
```

Comment:
stylish:
```
fun(...)
  when ... ->
    stuff;
```
is better then line wraps and definitely better then wrapping in the middle of the guards

Reply:
Got it. Fixed.